src/features/commands/command-line-hook.tsx

```diff
@@ -117,12 +125,62 @@ export const CommandLineComponent: React.FC<{
   const [selectedActionIndex, setSelectedActionIndex] = useState(0)
   const [firstActionIndex, setFirstActionIndex] = useState(0)
 
-  useEffect(() => {
+  const {
+    me: { email: myEmail },
+  } = useMe()
+  const { data } = useUsersQuery()
+  const [sharePath] = useSharePathMutation()
+
+  const handleShare = (user: Omit<User, 'notifications' | 'signature'>) => {
+    toast.promise(
+      sharePath({ variables: { path: location.pathname, userId: user.id } }),
+      {
+        loading: 'Sharing page',
+        success: `Page shared with ${user.fullName.split(' ')[0]}`,
+        error: 'Could not share page',
+      },
+    )
+  }
+
+  const advancedActions: CommandLineAction[] = [
+    {
+      label: 'Share path',
+      onResolve: () => setSearchValue('/share @'),
+    },
+  ]
+
+  const setUsersAsResult = () => {
+    const name = searchValue.split('@')[searchValue.split('@').length - 1]
+    const users =
+      data?.users?.filter(
+        (user) =>
+          user.email !== myEmail &&
+          user.fullName.toLowerCase().includes(name.toLowerCase()),
+      ) ?? []
+
     setSearchResult(
-      actions.filter((item) => {
-        return item.label.toLowerCase().includes(searchValue.toLowerCase())
-      }),
+      users?.map((user) => ({
+        label: user.fullName,
+        onResolve: () => {
+          handleShare(user)
+          setSearchValue(`/share @${user.fullName}`)
+        },
+      })) || [],
     )
+  }
+
+  useEffect(() => {
+    if (searchValue === '/') {
+      setSearchResult(advancedActions)
+    } else if (searchValue.includes('/share') && searchValue.includes('@')) {
+      setUsersAsResult()
+    } else {
```

Comment:
This will provide an "index out of bounds" error but the basic idea is there. This way you will get search help if you write "/sh"
```suggestion
    if (searchValue.includes('/share') && searchValue.includes('@')) {
      setSearchResult(advancedActions)
    } else if (searchValue[0] === '/') {
      setUsersAsResult()
    } else {
```